Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
`con·de·scend·ing
ˌkändəˈsendiNG/
adjective
having or showing a feeling of patronizing superiority`
(From Google)
See what I mean?
The same word could refer to the continuous of "to condescend", but also to an adjective, which has a different meaning in Spanish.

> preserve the original context...

What exactly do you understand by context?
Preserving context does NOT mean translate word by word. The context is the conceptual "environment" surrounding the text. Again, conceptual, not literal.

If we enforce to keep the word sequence so tight between languages, translations to both Spanish and Italian, or Chinese would be close to impossible.